Formula/pipenv.rb

```diff
@@ -0,0 +1,40 @@
+class Pipenv < Formula
+  include Language::Python::Virtualenv
+  desc "Sacred Marriage of Pipfile, Pip, & Virtualenv by @kennethreitz"
```

Comment:
Don't need `kennethreitz` in here. A better description would be good, not clear what this means.